<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BackupSettingsForm</class>
 <widget class="QDialog" name="BackupSettingsForm">
  <property name="windowTitle">
   <string>Настройки резервного копирования</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelPath">
     <property name="text"><string>Путь к резервной папке:</string></property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="backupPath"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelSchedule">
     <property name="text"><string>Интервал (мин):</string></property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="backupInterval">
     <property name="minimum"><number>5</number></property>
     <property name="maximum"><number>1440</number></property>
     <property name="value"><number>60</number></property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="buttonSave">
     <property name="text"><string>Сохранить</string></property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>